src/Symfony/Bridge/PhpUnit/Compat/Framework/BaseTestListener.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Compat\Framework;
+
+if (class_exists(PHPUnit\Framework\BaseTestListener::class)) {
```

Comment:
and if it was kept, there is a missing ``\`` here, meaning you check for ``Symfony\Bridge\PhpUnit\Compat\Framework\PHPUnit\Framework\BaseTestListener``